scripts.py

```diff
@@ -2010,64 +2011,6 @@ def process_item(self, collection, link_obj, policy):
         )
 
 
-class RefreshMaterializedViewsScript(TimestampScript):
-    """Refresh all materialized views."""
-
-    @classmethod
-    def arg_parser(cls):
-        parser = argparse.ArgumentParser()
-        parser.add_argument(
-            '--blocking-refresh',
-            help="Provide this argument if you're on an older version of Postgres and can't refresh materialized views concurrently.",
-            action='store_true',
-        )
-        return parser
-
-    def do_run(self):
-        args = self.parse_command_line()
-        if args.blocking_refresh:
-            concurrently = ''
-        else:
-            concurrently = 'CONCURRENTLY'
-
-        # Initialize the default database session. We can't use this
-        # session to run the VACUUM commands, because it wraps
-        # everything in a big transaction, and VACUUM can't be
-        # executed within a transaction block. But we will use it at
-        # the end, to recalculate the sizes of lanes.
-        db = self._db
-
-        url = Configuration.database_url()
-        engine = create_engine(url, isolation_level="AUTOCOMMIT")
-        engine.autocommit = True
-        a = time.time()
-        engine.execute("VACUUM (VERBOSE, ANALYZE)")
-        b = time.time()
-        self.log.info("Database vacuumed in %.2f sec." % (b-a))
-
-        for view_name in SessionManager.MATERIALIZED_VIEWS.keys():
-            a = time.time()
-            engine.execute("REFRESH MATERIALIZED VIEW %s %s" % (concurrently, view_name))
-            b = time.time()
-            self.log.info("%s refreshed in %.2f sec.", view_name, b-a)
-
-            # Vacuum the materialized view immediately after
-            # refreshing it.
-            a = time.time()
-            engine.execute("VACUUM ANALYZE %s" % view_name)
-            b = time.time()
-            self.log.info("%s vacuumed in %.2f sec", view_name, b-a)
-
-        # Recalculate the sizes of lanes.
-        for lane in db.query(Lane):
-            lane.update_size(db)
-            self.log.info(
-                "Size of lane %s calculated as %s",
-                lane.full_identifier, lane.size
-            )
-        db.commit()
```

Comment:
Until I get to https://jira.nypl.org/browse/SIMPLY-2107 we won't have any way of recalculating lane sizes.